app/src/main/java/ch/epfl/sweng/SDP/home/LeaderboardActivity.java

```diff
@@ -312,24 +311,25 @@ private void initLayout() {
             setTag("friendsButton" + index);
 
             // set friendsButton invisible to yourself
-            if(player.username.equals(Account.getInstance(context).getUsername())) {
+            if (player.username.equals(Account.getInstance(context).getUsername())) {
                 setVisibility(View.INVISIBLE);
             }
         }
 
         /**
-         * Gets data if users are friends, else null.
-         * Then applies listener.
-         * @param listener  how to handle response
+         * Gets data if users are friends, else null. Then applies listener.
+         *
+         * @param listener how to handle response
          */
         private void isFriendWithCurrentUser(ValueEventListener listener) {
-            Database.INSTANCE.getReference("users").child(player.userId).child("friends")
-                    .child(Account.getInstance(context).getUserId())
-                    .addListenerForSingleValueEvent(listener);
+            Database.constructBuilder().addChildren(
+                    format("users.%s.friends.%s", player.userId, Account.getInstance(context)
+                            .getUserId())).build().addListenerForSingleValueEvent(listener);
```

Comment:
Put .getUserId() on the same line as Account.getInstance(), since they belong toghether.

Reply:
I would like to, but then CodeClimate would probably complain about the line length!

Reply:
Then put Account.getInstance() on the new line 😁.